dev/make/compiler_definitions/dpcpp.mk

```diff
@@ -25,7 +25,7 @@ CMPLRDIRSUFF.dpcpp = _dpcpp
 
 CORE.SERV.COMPILER.dpcpp = generic
 
--Zl.dpcpp = $(if $(OS_is_win),-Zl,) $(-Q)no-intel-lib
+-Zl.dpcpp = $(if $(OS_is_win),-Zl -Q,-)no-intel-lib
```

Comment:
Any ideas why it doesnt work by default? The -Q flag define in the beginning of makefile, and in scenario when we build onedal with command make onedal_dpc -j n COMPILER=vc it should take these 
-Q     := $(if $(OS_is_win),$(if $(COMPILER_is_vc),-,-Q),-)

Reply:
discussed offline